### ISSUE TITLE
Broken internal dependencies in 2.0.1

### ISSUE THREAD

**Issue by jasonvasquez** [apollographql/apollo-server#1530]
- [ ] has-reproduction
- [ ] feature
- [ ] docs
- [x] blocking
- [ ] good first issue

I'm working specifically with `apollo-server-hapi`.   As of the 2.0.1 release that pushed to npm today (commit ef82add05b1dc7c28b4f8a2c5341cf81a7fc76f2), it depends on local file directories for dependencies in its package.json.

As a result:
```
❯ yarn install
yarn install v1.7.0
[1/5] Validating package.json...
[2/5] Resolving packages...
error Package "apollo-server-core" refers to a non-existing file '"<my-file-path/>\\apollo-server-core"'.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by mikebm** [comment]
Same issue here

**Comment by tobobo** [comment]
Installing `apollo-server-core@2.0.0` works fine, only the latest `apollo-server-core@2.0.1` is broken

**Comment by dnalborczyk** [comment]
with `npm`: [2.0.1 upgrade from 2.0.0]

```shell
npm ERR! path /api/node_modules/apollo-server-core
npm ERR! code ELOOP
npm ERR! errno -62
npm ERR! syscall access
npm ERR! ELOOP: too many symbolic links encountered, access '/api/node_modules/apollo-server-core'
```

**Comment by obsessen** [comment]
I have the same problems while installing vue-cli which has it as a dependancy.

Yarn:
```
[1/4] 🔍  Resolving packages...
error Package "apollo-server-core" refers to a non-existing file '"~/.config/yarn/apollo-server-core"'.
info Visit https://yarnpkg.com/en/docs/cli/global for documentation about this command.
```

npm:
```
npm ERR! code ENOLOCAL
npm ERR! Could not install from "../../usr/local/lib/node_modules/@vue/cli/node_modules/apollo-server-core" as it does not contain a package.json file.

npm ERR! A complete log of this run can be found in:
npm ERR!    ~/.npm/_logs/2018-08-14T18_37_09_923Z-debug.log
```

**Comment by chroriginal** [comment]
+1

**Comment by mikebm** [comment]
Looks to be fixed with 2.0.2.

**Comment by jasonvasquez** [comment]
Thanks! Looks like 2.0.2 is working for me too
